geometry_msgs/Point position
bool is_claimed
int32 target_id